Modules/FV0/src/DigitQcTask.cxx

```diff
@@ -641,6 +656,18 @@ void DigitQcTask::endOfCycle()
   ILOG(Debug, Support) << "adding last TF creation time: " << mTFcreationTime << ENDM;
   getObjectsManager()->getMonitorObject(mHistBCvsTrg->GetName())->addOrUpdateMetadata("TFcreationTime", std::to_string(mTFcreationTime));
 
+  for (int channel = 1; channel <= sNCHANNELS_FV0_PLUSREF-1; channel++) {
+    int events_in_range = 0;
+    int events_per_channel = 0;
+    for (int bin_on_y_axis = 1; bin_on_y_axis <= mHistTime2Ch->GetNbinsY(); bin_on_y_axis++) {
+        if (mHistTime2Ch->GetYaxis()->GetBinLowEdge(bin_on_y_axis) >= -mTimeGate && mHistTime2Ch->GetYaxis()->GetBinUpEdge(bin_on_y_axis) <= mTimeGate) {
+            events_in_range += mHistTime2Ch->GetBinContent(channel, bin_on_y_axis);
+        }
+        events_per_channel += mHistTime2Ch->GetBinContent(channel, bin_on_y_axis);
+    }
+    mHistGateTimeRatio2Ch->SetBinContent(channel, (float) events_in_range / (float) events_per_channel);
+  }
```

Comment:
Why don't you calculate this at Post-Processing stage(which at remote QC node in production)? I'm asking because DigitQCtask is already heavy, so it is better to distribute calculations among different sections. 
Also there might be zero division error at runtime `(float) events_in_range / (float) events_per_channel`. You can use hist projections for such calculations. Here some example which we preliminary implemented for FT0 https://github.com/AliceO2Group/QualityControl/blob/8dc17a43438bf47b2a3ad7d499c6c0dd6f8036d9/Modules/FT0/src/PostProcTask.cxx#L315-L321

Reply:
@afurs @DawidSkora @andreasmolander we followed the CDF_efficiency. It is a good idea to move it to PostProc but one could also make it for the CFDefficiency. We should decide it asap.

Reply:
@afurs Side remark. I see +/-190 fix value in FT0.  As far as I remember it should be 192 and set as parameter.